### PR TITLE
8 create job postings backend

### PR DESCRIPTION
**Pull request by ArielleWaks**
#### As a user I want to create a job posting so that I can look for someone to watch my dog on specific dates by providing my name, location, contact info, pet name, pet type, and price.
- [x] Backend:
    - [x] Setup unit test for Job model https://courses.baeldung.com/courses/1295711/lectures/30127904 **_first draft_**
    -  ~~Setup unit test for Job controller~~
    - [x] Postman project **_first draft_**
    - [x] Create model **_first draft_**
    - [x] Create controller **_first draft_**
    - [x] In memory H2 database

### CODE REVIEW

File: backend/src/main/java/org/launchcode/fancyrats/models/Job.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    //final?
+    private LocalDate createdDate;
+
+    private long payRate;
+
+    private String description;
+
+    @ManyToOne
+    private Client client;
```
Comment (ArielleWaks) [inline backend/src/main/java/org/launchcode/fancyrats/models/Job.java]: Hey @Bryanary-code, let me know what you're thinking about for the user model. I'm adding a comment here so I remember where to swap it out of the Job model :)

File: backend/src/main/java/org/launchcode/fancyrats/models/Job.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    private LocalDate endDate;
+
+    //final?
+    private LocalDate createdDate;
+
+    private long payRate;
+
+    private String description;
+
```
Comment (ArielleWaks) [inline backend/src/main/java/org/launchcode/fancyrats/models/Job.java:43]: First draft of Job fields does anybody have any thoughts/suggestions?

File: backend/src/main/java/org/launchcode/fancyrats/models/Job.java (hunk 1 above)
Comment (Bryanary-code) [reply backend/src/main/java/org/launchcode/fancyrats/models/Job.java]: Hey, I'm thinking:  id
                              username
                              password
                              email
                              name 
                              address
                              phone number
                              role (pet sitter, pet owner, both)
                              pet type(s)
I think these should suffice for a basic user profile. Then allow for more customization after account creation ( i.e. photos, bio, reviews, social links, etc.) What do you think?

File: backend/src/main/java/org/launchcode/fancyrats/models/Job.java (hunk 2 above)
Comment (ArielleWaks) [reply backend/src/main/java/org/launchcode/fancyrats/models/Job.java:43]: @byersj920 here are the fields so far

File: backend/build.gradle [hunk 3]
```diff
@@ -21,31 +18,27 @@ java {
...
     annotationProcessor 'org.projectlombok:lombok'
     asciidoctorExt 'org.springframework.restdocs:spring-restdocs-asciidoctor'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.restdocs:spring-restdocs-mockmvc'
-    implementation 'jakarta.validation:jakarta.validation-api:3.0.0'
-    implementation 'org.springframework.boot:spring-boot-starter-security'
-    implementation 'io.jsonwebtoken:jjwt-api:0.11.2'
-    implementation 'io.jsonwebtoken:jjwt-impl:0.11.2'
-    implementation 'io.jsonwebtoken:jjwt-jackson:0.11.2'
-    compileOnly 'org.apache.tomcat.embed:tomcat-embed-websocket'
```
Comment (ArielleWaks) [inline backend/build.gradle:43]: Hey @Bryanary-code I removed the tomcat dependency and it works for me but I can definitely put it back I know tomcat was giving you issues

File: backend/build.gradle [hunk 4]
```diff
@@ -3,13 +3,10 @@ plugins {
     id 'org.springframework.boot' version '3.2.3'
     id 'io.spring.dependency-management' version '1.1.4'
     id 'org.asciidoctor.jvm.convert' version '3.3.2'
-    id 'java-library'
-    id 'maven-publish'
```
Comment (ArielleWaks) [inline backend/build.gradle:7]: @Bryanary-code we can add maven-publish and java-library back in if you need them.

**Review by byersj920 — APPROVED** [review]

**Pull request merged**